debian/rules

```diff
@@ -1,100 +1,84 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-#
-# This file was originally written by Joey Hess and Craig Small.
-# As a special exception, when this file is copied by dh-make into a
-# dh-make output file, you may use that output file without restriction.
-# This special exception was added by Craig Small in version 0.37 of dh-make.
-#
-# Modified to make a template file for a multi-binary package with separated
-# build-arch and build-indep targets  by Bill Allombert 2001
 #export DH_VERBOSE=1
 
-PACKAGE=openvswitch
-PACKAGE_DKMS=openvswitch-datapath-dkms
-include /usr/share/dpkg/pkg-info.mk
-
-ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
-PARALLEL = -j$(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
-else
-PARALLEL =
-endif
-
 %:
-	dh $@ --with autoreconf,python2 --parallel
+	dh $@ --with autoreconf,python3,systemd,sphinxdoc
 
 # use --as-needed only if supported by dh-autoreconf (to simplify backporting)
 DH_AS_NEEDED=$(shell dpkg --compare-versions $$(dpkg --status dh-autoreconf | grep Version | cut -d' ' -f2) ge 6 && echo --as-needed)
 override_dh_autoreconf:
 	dh_autoreconf $(DH_AS_NEEDED)
 
+ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
+PARALLEL = -j$(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
+else
+PARALLEL =
+endif
+
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-ssl --enable-shared $(DATAPATH_CONFIGURE_OPTS)
+	# Configure OVS before OVN
+	test -d $(CURDIR)/ovs || mkdir ovs
+	cd ovs && tar -xzf /usr/src/openvswitch/openvswitch.tar.gz --strip-components=1
+	cd ovs && ./configure --enable-ssl --localstatedir=/var --sysconfdir=/etc --prefix=/usr
+	# Use dh_auto_configure to build OVN
+	dh_auto_configure -- \
+		--prefix=/usr \
+		--localstatedir=/var \
+		--sysconfdir=/etc \
+		--with-dbdir=/var/lib/ovn \
+		--with-ovs-source=$(CURDIR)/ovs \
```

Comment:
Previous varian was more flexible, I think, it will be better to use $(DATAPATH_CONFIGURE_OPTS) for parameters defining
```
RUN DATADIR="/usr/share/ovn" \
    DATAPATH_CONFIGURE_OPTS="--with-ovs-source=${OVS_SRC} --with-ovs-build=${OVS_SRC}" \
    DEB_BUILD_OPTIONS="parallel=8 nocheck" \
    fakeroot debian/rules binary
```